x-pack/plugins/canvas/shareable_runtime/webpack/ci_stats_plugin.ts

```diff
@@ -41,7 +41,7 @@ export class CiStatsPlugin {
       return;
     }
 
-    compiler.hooks.emit.tapAsync('CiStatsPlugin', async (compilation) => {
+    compiler.hooks.emit.tapPromise('CiStatsPlugin', async (compilation) => {
```

Comment:
Nice find! I wonder if I've made this mistake in other places...

Reply:
:partying_face: I reopened this change at https://github.com/elastic/kibana/pull/137972 to avoid partial backport confusion